ide/rust/RepoCargoManifestGenerator.kt

```diff
@@ -95,11 +98,16 @@ class RepoCargoManifestGenerator(private val repository: File, private val bazel
 
     private inner class TargetCargoManifestGenerator(private val info: TargetSyncInfo) {
         fun generateManifest(): File {
+            if (info.containsGeneratedSources) buildTarget()
```

Comment:
If the target has generated sources, build it to generate those sources so that `cargo build` can subsequently succeed.

This enables IDE tooling and debugging.